services/src/main/java/org/apache/unomi/services/services/ProfileServiceImpl.java

```diff
@@ -997,6 +997,9 @@ public void bundleChanged(BundleEvent event) {
     private boolean merge(Map<String, Object> target, Map<String, Object> object) {
         boolean changed = false;
         for (Map.Entry<String, Object> newEntry : object.entrySet()) {
+
+            String packageName = newEntry.getValue().getClass().getPackage().getName();
```

Comment:
@micheleriva if `newEntry.getValue()` is null, it will throw a NPE.

Reply:
Hi @fpapon 	,
I actually used the original Unomi code (see [here](https://github.com/apache/unomi/blob/master/services/src/main/java/org/apache/unomi/services/services/ProfileServiceImpl.java#L1012)), in order to retrive that value, I didn't want to upset the codebase

Reply:
@micheleriva yes but it's better to put this line after the test `if (newEntry.getValue() != null) {`
=> https://github.com/apache/unomi/pull/92/files#diff-b144601886f209ff7ba78b31f594f199R1003

Reply:
NPE is tested after so in the actual code it works.
=>https://github.com/apache/unomi/pull/92/files#diff-b144601886f209ff7ba78b31f594f199R1003

Reply:
Ok I get your point, my bad. I'm sending an updated commit, thanks for your review

Reply:
Great ;) Thanks for your PR!